core/src/main/java/io/grpc/Attributes.java

```diff
@@ -31,9 +31,13 @@
 
 package io.grpc;
 
+import static com.google.common.collect.Maps.newHashMap;
```

Comment:
Note that this means it can be okay to use collect in our Netty project, since that isn't intended for Android.